plugins/path/plugin.zsh

```diff
@@ -70,5 +72,12 @@ geometry_prompt_path_render() {
     dir=$(basename $PWD)
   fi
 
-  echo "$GEOMETRY_PROMPT_PREFIX$GEOMETRY_PROMPT_PREFIX_SPACER%${#PROMPT_SYMBOL}{%(?.$GEOMETRY_PROMPT.$GEOMETRY_EXIT_VALUE)%}$GEOMETRY_SYMBOL_SPACER%F{$GEOMETRY_COLOR_DIR}$dir%f$GEOMETRY_DIR_SPACER$GEOMETRY_PROMPT_SUFFIX"
+  local prompt_prefix="$GEOMETRY_PROMPT_PREFIX$GEOMETRY_PROMPT_PREFIX_SPACER"
+
+  local symbol_width="${#${(S%%)prompt_symbol//\\e\[[0-9]m}}"
```

Comment:
Can you add a comment linking to this issue
And mayb a TODO/FIXME comment about the more generic solution?
`local symbol_width="${#${(S%%)prompt_symbol//(\%([KF1]|)\{*\}|\%[Bbkf])}}"`

Reply:
I'd actually prefer that we encourage the use of `%B` for bold, since it goes better with a more generic solution, like the `${(S%%)prompt_symbol//(\%([KF1]|)\{*\}|\%[Bbkf])}` substitution. Updating the PR to use that instead and linking the issue.